src/printer/enum_to_string.h

```diff
@@ -0,0 +1,45 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2024 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Utilities for printing API enum values.
+ *
+ * In particular, we require `toString(...)` to be defined for any identifier
+ * that may appear in statistics, for safe printing (src/util/safe_print.h).
```

Comment:
So far only skolem id are included in this file. So statistics here seems out of place. I guess all enum printers would be added here.

Reply:
Yes, I discussed this with @aniemetz offline. We decided to add this file and gradually migrate the API enum values that require both `std::to_string` and `toString` to this header.